index.html

```diff
@@ -71,8 +71,8 @@ <h3 class="display-4">Laminas Documentation</h3>
 
             <div class="row row-cols-1 row-cols-md-4">
                 <div class="col mb-2">
-                    <a href="https://docs.mezzio.dev/" class="mezzio">
-                        <div class="card h-100 text-center">
+                    <div class="card h-100 text-center">
+                        <a href="https://docs.mezzio.dev/" class="mezzio">
```

Comment:
With your changes we also must move the class of the subproject to the `div.card` element otherwise we loose the coloured top border.

Reply:
Sorry for that. I must be more careful next time. The PR seems to have been updated automatically.

Reply:
> Sorry for that. I must be more careful next time.

No problem! Every support is welcome.